### PR TITLE
Add a section with information about the project

### PR DESCRIPTION
**Pull request by AdamSzL**
I created a section with a bit of information about the project, for example a link to the github repository

### CODE REVIEW

**Comment by AdamSzL** [comment]
Waiting for a review

File: src/components/EnterRoomPage.tsx [hunk 1]
```diff
@@ -13,19 +13,26 @@ function EnterRoomPage({changeStage}: EnterRoomPageInterface) {
...
+                    />
+                    <button className="bg-green-500 hover:bg-green-600 text-white px-2 py-1 rounded text-2xl" onClick={()=>changeStage(1)}>Play!</button>
+                    <button className="bg-blue-500 hover:bg-blue-600 text-white px-2 py-1 rounded text-xl" onClick={joinPrivateRoom}>Create Private Room</button>
+                </div>
+            </div>
+            <div className="bg-gray-700 w-full flex-1 text-center">
+                <p className="text-2xl text-white text-center my-5">About the project</p>
+                <p className="text-white my-3 mx-2">A web project representing the well-known online game - charades. A canvas-based game where you can create a room and play online with your friends!</p>
+                <a href="https://github.com/AGH-Narzedzia-Informatyczne-2023-2024/charades-online" className="text-blue-200 hover:underline" target="_blank">GitHub repository</a>
             </div>
```
Comment (mJastrzebski6) [inline src/components/EnterRoomPage.tsx:36]: good change

**Review by mJastrzebski6 — APPROVED** [review]
not a big update but very useful

File: src/components/EnterRoomPage.tsx (hunk 1 above)
Comment (AdamSzL) [reply src/components/EnterRoomPage.tsx:36]: Thanks! This section will also be updated in the future

**Pull request merged**